src/i18n/resources/en.js

```diff
@@ -1050,6 +1050,7 @@ The FRA team
       'The sum of the reported FRA categories exceeds the forest area reported in table 1a',
     valueEqualToSum: 'Total value should be equal to sum of subcategories',
     landAreaExceedsTotalLandArea: 'Land area exceeds total land area (1a)',
+    classValueNotGreaterThan: '{{name}} should be not greater than {{value}}',
```

Comment:
`{{value}} %`

also can be directly 100%? it's always 100

Reply:
In the video I noticed that 're' is only 10% but error shows 'greater than 100'
Should it be: class should total to 100%?

Reply:
I would like to keep it dynamic, sooner or later I would like reduce all the error messages to a small bunch of dynamic ones.

<img width="1070" alt="Screenshot 2023-03-01 alle 13 40 13" src="https://user-images.githubusercontent.com/10047945/222142167-7898c563-2d4e-4ebf-8bbd-3b1b0243c66a.png">

Reply:
'to 100%'